webpack.config.js

```diff
@@ -86,7 +86,8 @@ module.exports = [
                 'scratch-storage',
                 // Renderer
                 'scratch-render'
-            ]
+            ],
+            'motion-extension': './src/extensions/scratch3_video_sensing/debug'
```

Comment:
It shouldn't be necessary to explicitly mention the sensing extension in the webpack config as long as it's `require`d as you do in `extension-manager.js`. If you want to expose debug stuff to the JS console, I would use `global` (which Webpack turns into `self` or `window`). I imagine you're familiar, but if not, you can find an example in `src/extension-support/extension-worker.js`.

Reply:
This is used by `src/playground/motion.html`.